hypothesis-python/tests/common/debug.py

```diff
@@ -29,14 +29,13 @@ class Timeout(BaseException):
 
 def minimal(definition, condition=lambda x: True, settings=None, timeout_after=10):
     def wrapped_condition(x):
-        if timeout_after is not None:
-            if runtime:
-                runtime[0] += TIME_INCREMENT
-                if runtime[0] >= timeout_after:
-                    raise Timeout()
         result = condition(x)
         if result and not runtime:
             runtime.append(0.0)
+        if timeout_after is not None and runtime:
```

Comment:
The reason this moved here is because in the feature flags tests we do data generation in the condition itself, and if the timeout runs before the condition then that raises a `Flaky` error.